cpu/lpc1768/periph/uart.c

```diff
@@ -154,6 +154,8 @@ void uart_write(uart_t uart, const uint8_t *data, size_t len)
             break;
 #endif
         default:
+            /* prevent cppcheck uninitialized variable error */
+            dev = (LPC_UART_TypeDef *) 0;
```

Comment:
sorry for delaying this. However, I would suggest to simply init `LPC_UART_TypeDef *dev = NULL;` above.

Reply:
@jia200x what do you think?